flight/Modules/Actuator/actuator.c

```diff
@@ -686,6 +692,11 @@ static bool set_channel(uint8_t mixer_channel, uint16_t value, const ActuatorSet
 		case ACTUATORSETTINGS_CHANNELTYPE_PWM:
 			PIOS_Servo_Set(actuatorSettings->ChannelAddr[mixer_channel], value);
 			return true;
+#if defined(PIOS_INCLUDE_ONESHOT)
```

Comment:
I think you need to move this conditional down a line. In the case oneshot is not included, we should probably do something like set an error to prevent arming. It might also make sense to set PIOS_Servo to 0 in this case to make sure the output is low.